types/jsreport-core/index.d.ts

```diff
@@ -15,20 +18,36 @@ declare namespace JsReport {
     type Recipe = "html";
 
     interface Template {
-        content: string;
+        content?: string;
         engine: Engine | string;
-        helpers: Helpers;
+        helpers?: Helpers;
```

Comment:
`?` is not needed here, too.

Reply:
without the ? one would be required to define "helpers" when using in this [manor](https://github.com/newbish/DefinitelyTyped/blob/a9c6cb9c7c9ed57c332cefa4e19061d8768c4c59/types/jsreport-core/jsreport-core-tests.ts#L3-L7).

Reply:
I don't believe _helpers_ are required.